content/doc/book/pipeline/index.adoc

```diff
@@ -6,43 +6,150 @@ layout: chapter
+node { // <1>
+    stage('Build') { // <2>
+        sh 'make' // <3>
+    }
+
+    stage('Test') {
+        sh 'make check'
+        junit 'reports/**/*.xml' // <4>
+    }
+
+    stage('Deploy') {
+        sh 'make publish'
+    }
+}
+----
+<1> <<node,`node`>> indicates that Jenkins should allocate an executor and workspace for
+this part of the Pipeline.
+<2> <<stage,`stage`>> describes a stage of this Pipeline.
+<3> `sh` executes the given shell command
+<4> `junit` is a Pipeline <<step,step>> provided by the
+link:https://plugins.jenkins.io/junit[JUnit plugin]
+for aggregating test reports.
+
+[[why]]
+== Why Pipeline?
+
+Jenkins is, fundamentally, an automation engine which supports a number of
+automation patterns. Pipeline adds a powerful set of automation tools onto
+Jenkins, supporting use-cases that span from simple continuous integration to
+comprehensive continuous delivery pipelines. By modeling a series of related
+tasks, users can take advantage of the many features of Pipeline:
+
+* *Code*: Pipelines are implemented "as code," and typically checked into
+  source control, giving teams the abilit to edit, review, and iterate upon
+  their delivery pipeline.
+* *Durable*: Pipelines can survive both planned, and unplanned, restarts of the
+  Jenkins master.
+* *Pausable*: Pipelines can optionally stop and wait for human input or approval
+  before continuing the Pipeline run.
+* *Versatile*: Pipelines support complex real-world continuous delivery
+  requirements, including the ability to fork/join, loop, and perform work in
+  parallel.
+* *Extensible*: The Pipeline plugin supports custom extensions to its DSL
+  footnoteref:[dsl]
+  and multiple options for integration with other plugins.
+
+
+While Jenkins has always allowed rudimentary forms of chaining Freestyle Jobs
+together to perform sequential tasks,
+footnote:[Additional plugins have been used to implement complex behaviors
+utilizing Freestyle Jobs such as the Copy Artifact, Parameterized Trigger,
+and Promoted Builds plugins]
+Pipeline makes this concept a first-class citizen in Jenkins.
+
+Building on the core Jenkins value of extensibility, Pipeline is also
+extensible both by users with <<pipeline/shared-libraries#,Pipeline Shared Libraries>>
+and by plugin developers.
+footnoteref:[ghof,link:https://plugins.jenkins.io/github-organization-folder[GitHub
+Organization Folder plugin]]
+
+
+The flowchart below is an example of one continuous delivery scenario easily
+modeled in Jenkins Pipeline:
+
+image::/images/pipeline/realworld-pipeline-flow.png[title="Pipeline Flow", 800]
+
+
+[[terms]]
+== Pipeline Terms
 
-If you are a Jenkins administrator and want to know more about managing Jenkins nodes and instances, see
-<<managing#,Managing Jenkins>>.
+[[step]]
+Step::
+    A single task, fundamentally steps tell Jenkins _what_ to do. For example, in
+    order to instruct Jenkins to execute the shell command `make`, the `sh`
+    step would be used: `sh 'make'`.
```

Comment:
This sentence doesn't read right to me.

Change 'the sh step' to 'the following sh step' or 'this sh step' or 'the sh step used would be'?